community/kernel/src/main/java/org/neo4j/kernel/impl/factory/EditionModule.java

```diff
@@ -117,19 +130,39 @@ public static AuthManager createAuthManager( Config config, LogService logging )
         }
 
         String key = config.get( GraphDatabaseSettings.auth_manager );
+        Log authManagerLog;
+        try
+        {
+            authManagerLog = authManagerLog( config, fileSystem, jobScheduler );
+        }
+        catch ( IOException ioe ){
+            logging.getInternalLog( GraphDatabaseFacadeFactory.class ).warn( "Unable to create log for auth-manager. " +
+                    "Auth logging turned off." );
+            authManagerLog = NullLog.getInstance();
+        }
```

Comment:
This could also be a `createSecurityLog` (`createAuthManagerLog`) method, and then we can just give the log it returns as a parameter to `createAuthManager`.
(Although I suspect we may soon need both dependencies (file system, job scheduler) in auth manager anyways, comment is about as this PR stands on its own)